### ISSUE TITLE
Running a complex table component created this error

### ISSUE THREAD

**Issue by robbyhsu** [EightShapes/specs-plugin#86]
### Description

I ran a complex table component on this which had multiple sub-components
Link to the component:
https://www.figma.com/file/g4sX3uCO8fAyDtcLC4SEl3/Untitled?type=design&node-id=1%3A93343&t=INtbxYO3FuDvzFrv-1

### Steps to reproduce
Component Screenshot
![image](https://github.com/EightShapes/specs-plugin/assets/11588687/b7701972-bc73-430e-9b16-2ede78b0805e)

{Add steps to reproduce the error}

![image](https://github.com/EightShapes/specs-plugin/assets/11588687/b3bbe6a0-4d2e-41ed-b93e-5340d661a642)

{If applicable, add screenshots such as the Figma canvas, layer panel or design panel}

### Additional context

{Add any other context about the problem here}

### Defect log

{Copy the message from the browser/Figma console, if relevant}

**Comment by nathanacurtis** [comment]
@robbyhsu Yeah, I could see where running specs on things like this would run out of memory, and it's not something the plugin itself could resolve. In my clients' projects, tables like these...
* Include many, many repetitive rows
* Include many cells per row
* Include many _hidden_ cells per row
* Per cell include many _hidden_ elements
The depth and complicatedness of layers becomes something that chokes the plugin trying to COMPARE those things across the myriad props you have have displayed. A calendar picker has a similar challenge, redundantly running specs for 31 "day" cells when you really only need to spec one of those days.

To use the specs plugin effectively in this situation, consider...
* Running the plugin on modularly spec smaller pieces, such as "just one row"
* Reduce the number of rows and / or cells in your component so that you aren't redundantly spec'ing hundreds if not thousands of items 
* Detach and remove what may be many, many hidden layers that are irrelevant to the spec
